helm/ako/values.yaml

```diff
@@ -8,26 +8,18 @@ image:
   repository: 10.79.172.11:5000/avi-buildops/ako
   pullPolicy: IfNotPresent
 
-configs:
-  controllerVersion: "18.2.8"
-  shardVSSize: "LARGE"
-  passthroughShardSize: "SMALL"
-  fullSyncFrequency: "300"
-  cloudName: "Default-Cloud"
-  clusterName: ""
-  defaultDomain: ""
-  disableStaticRouteSync: "false"
-  ingressApi: "corev1" #enum: extensionv1 and corev1 - if not set default to corev1
-  defaultIngController: "true"
-  subnetIP: "" # Subnet IP of the data network
-  subnetPrefix: "" # Subnet Prefix of the data network
-  networkName: "" # Network Name of the data network
-  l7ShardingScheme: "hostname"
-  cniPlugin: "" #enum: calico|canal|flannel|openshift
-  logLevel: "INFO" #enum: INFO|DEBUG|WARN|ERROR
-  deleteConfig: "false" # Has to be set to true in configmap if user wants to delete AKO created objects from AVI
-  serviceType: ClusterIP #enum NodePort|ClusterIP
-  serviceEngineGroupName: "Default-Group" # Name of the ServiceEngine Group.
+### This section outlines the generic AKO settings
+AKOSettings:
+  logLevel: "WARN" #enum: INFO|DEBUG|WARN|ERROR
```

Comment:
Should we not retain INFO as default?

Reply:
We could, I just thought we should aggressively write code to detect problems through WARN messages. Maybe we will do in some more time. I will change it to INFO.